docs/pcp-programmers-guide.rst

```diff
@@ -0,0 +1,7966 @@
+========================================
+Performance Co-Pilot™ Programmer's Guide
+========================================
+
+.. _id5178752:
```

Comment:
What are these markers?

Reply:
The markers were created for link reference, however, it seems that the markers don't work well in Read the Docs. I would remove the markers soon.